incubator/hnc/internal/reconcilers/object.go

```diff
@@ -382,6 +388,24 @@ func (r *ObjectReconciler) syncPropagated(ctx context.Context, log logr.Logger,
 	return actionNop, nil
 }
 
+func (r *ObjectReconciler) getSelector(ctx context.Context, log logr.Logger, inst *unstructured.Unstructured) labels.Selector {
```

Comment:
`ctx` doesn't appear to be used in this function?

Reply:
It's also not used in `syncPropagated`, so I thought I need to keep it even though it's not used directly

Reply:
Maybe it used to be used there but was removed. I'd take it out of any place it's not (currently) needed, it's easy to add it back.